dipy/viz/tests/test_visualization.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
```

Comment:
@Tingyiwanyan If this a new functionality you are proposing which I think it is, then the function should go in the folder data/viz/ and not the tests folder. The data/viz/tests folder is for providing the functional tests using nosetests which you will have to do for your proposed function as well. Secondly, the tutorial you have provided should be renamed to some other name (check other examples). If this function is a work in progress then add the WIP tag to your PR. :)